pallets/moonbeam-lazy-migrations/src/lib.rs

```diff
@@ -106,5 +117,48 @@ pub mod pallet {
 			)
 			.into())
 		}
+
+		// TODO(alexandru): This extrinsic should be removed once Gov V1 is removed.
+		// Note: We don't need to unreserve any funds, as they are assumed to be already
+		// unreserved prior to this operation and the proposal submission disabled.
+		#[pallet::call_index(1)]
+		#[pallet::weight(
+			Weight::from_parts(0,
+				(MAX_BALANCES_LOCKS_STORAGE_ENTRY_SIZE + MAX_DEMOCRACY_VOTINGOF_STORAGE_ENTRY_SIZE) * <u64>::from(*limit))
```

Comment:
You should also account for intermediates nodes, it might be several bytes, I recommend to add a constant 4096 to the prrof size part of the weight hint